dask/dataframe/tests/test_arithmetics_reduction.py

```diff
@@ -1152,6 +1152,9 @@ def test_reductions_frame(split_every):
     assert_dask_graph(ddf1.mean(axis=1, split_every=split_every), "dataframe-mean")
 
 
+@pytest.mark.filterwarnings(
+    "ignore:Dropping of nuisance columns:FutureWarning"
+)  # https://github.com/dask/dask/issues/7714
```

Comment:
Just to confirm, we can remove this warning filter once we've implemented `numeric_only=` as mentioned over in https://github.com/dask/dask/issues/7714?

Reply:
Yeah exactly.  So filter the warning for now, and hopefully implement soon. Following the pattern in https://github.com/dask/dask/issues/7100